esi_leap/conf/api.py

```diff
@@ -23,6 +23,8 @@
     cfg.StrOpt('default_resource_type', default='ironic_node'),
     cfg.IntOpt('max_lease_time', default=21),
     cfg.IntOpt('default_lease_time', default=7),
+    cfg.StrOpt('SMTP_server', default='localhost'),
+    cfg.StrOpt('email_sender', default='admin@esi.massopen.cloud'),
```

Comment:
I don't think we want a default to be MOC-specific; we maybe shouldn't have a default for this at all.

It may make sense to include options for

* whether emails should be sent at all
* for the "lease expires in X days", a configuration for X

Reply:
Yep! I removed the email_sender default value and add lease_warning_days and enable_email configurations